test/dlc_tests/sanity/test_generate_coverage_doc.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import pytest
+from invoke.context import Context
+
+from test.test_utils import TEST_COVERAGE_FILE
+
+
+@pytest.mark.integration("Generating this coverage doc")
+def test_generate_coverage_doc():
+    """
+    Test generating the test coverage doc
+    """
+    test_coverage_file = TEST_COVERAGE_FILE
+    ctx = Context()
+    # Set DLC_TESTS to 'test' to avoid image names affecting function metadata (due to parametrization)
+    # Set CODEBUILD_RESOLVED_SOURCE_VERSION to test for ease of running this test locally
+    ctx.run("export DLC_TESTS='test' && export CODEBUILD_RESOLVED_SOURCE_VERSION='test' "
```

Comment:
nit: I can remove CODEBUILD_RESOLVED_SOURCE_VERSION here -- mostly did this to facilitate local testing, but we have a requirement to set this env variable set in our docs anyway